samples/sql-chatbot/src/main/java/io/quarkiverse/langchain4j/sample/chatbot/Movie.java

```diff
@@ -6,51 +6,63 @@
 import jakarta.persistence.Id;
 import jakarta.persistence.Table;
 
+import org.apache.camel.dataformat.bindy.annotation.CsvRecord;
+import org.apache.camel.dataformat.bindy.annotation.DataField;
 import org.hibernate.annotations.Comment;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 @Entity
 @Table(name = "movie", schema = "public")
+@CsvRecord(separator = ",", skipFirstLine = true )
 public class Movie {
 
     @Id
     @GeneratedValue
     private int id;
 
     @Column(name = "index")
+    @DataField(pos = 1)
```

Comment:
Does camel not support csv column names ? Seems redundant having to specify order here ?

Reply:
According to the [doc](https://camel.apache.org/components/4.8.x/dataformats/bindy-dataformat.html) : Camel Bindy doesn't. And it's not related to persistence so I don't know exactly if we can improve that